nationalarchives/ds-wagtail
# DF-767: Allow management of feedback submissions within Wagtail

Ticket URL: https://national-archives.atlassian.net/browse/DF-767

## About these changes

Ditches the custom report view for accessing feedback submission data in favour of registering the model as a Snippet - allowing those with the relevant model permissions to delete entries individually or in bulk.

## How to check these changes

In Wagtail, look for the 'Feedback' menu item in the left-hand menu and click it. From there, you should be able to:
- Export the data to XLSX/CSV (as you could previously in the custom report)
- Delete an individual entry
- Delete entries in bulk

Users need the 'Delete' permission for the FeedbackSubmission model in order for delete options to appear/work. This part is built-in Wagtail functionality - but, if you want to check it, you can create a group with "Access wagtail admin" and "Edit feedback submission" permissions, add a non-admin user to that group, then log in as that user to check that delete options do not work.


## Before assigning to reviewer, please make sure you have


## Merging PR guidance

Follow [docs\developer-guide\contributing.md](https://nationalarchives.github.io/ds-wagtail/developer-guide/contributing/)

## Deployment guidance

Follow [docs\infra\environments.md](https://nationalarchives.github.io/ds-wagtail/infra/environments/)
